build.sbt

```diff
@@ -5,6 +5,7 @@ inThisBuild(
       else "SNAPSHOT" // only for local publishng
     },
     scalaVersion := V.scala212,
+    crossScalaVersions := List(V.scala212),
```

Comment:
Do we need this option at all if we're not cross-building?

Reply:
I added it to ensure `+compile` does the right thing, just in case

Reply:
I don't know what `+compile` does when crossScalaVersions is unset